pytest/tests/contracts/infinite_loops.py

```diff
@@ -0,0 +1,64 @@
+# Spins up four nodes, deploy an smart contract to one node,
+# Call a smart contract method in another node
+import sys, time
+import base58
+import concurrent.futures
+import requests
+
+sys.path.append('lib')
+from cluster import start_cluster
+from transaction import sign_deploy_contract_tx, sign_function_call_tx
+from utils import load_binary_file, compile_rust_contract
+
+nodes = start_cluster(
+    4, 0, 1, None,
+    [["epoch_length", 10], ["block_producer_kickout_threshold", 80], ["transaction_validity_period", 10000]], {})
+
+wasm_file = compile_rust_contract(''' metadata! {
```

Comment:
Using nearsdk just to make a looping contract is an overkill. A specific issue here is that this test will be slower to run as it'll need to invoke cargo machinery to compile the contract. 

Instead, I suggest adding:

```rust
#[no_mangle]
pub unsafe fn loop_forever() {
    loop {}
}
```

To the canonical test contract: 

https://github.com/near/nearcore/blob/7e24d5d758be41ff12528964ebdcad12e32b990d/runtime/near-test-contracts/test-contract-rs/src/lib.rs#L561-L575

See this example for how to use this contract in a python test

https://github.com/matklad/nearcore/blob/21a32edde770aa3b8e4a14d78ab2a0bb707442be/pytest/tests/contracts/deploy_call_smart_contract.py#L21-L23/